lib/standard/numeric.nit

```diff
@@ -30,9 +30,11 @@ redef class Text
 	# assert "0.0".to_n == 0.0
 	# assert ".12345".to_n == 0.12345
 	# assert "12345".to_n == 12345
+	# assert "".to_n == 0
 	# ~~~~
 	fun to_n: Numeric
 	do
+		if is_empty then return 0
```

Comment:
Ow yeah, my bad, I changed the behaviour with #1588 :/

This should definitely not crash with this kind of error though, I'll check